Source/WebCore/platform/RectEdges.h

```diff
@@ -73,6 +73,21 @@ template<typename T> class RectEdges {
     void setBottom(const T& bottom) { setAt(BoxSide::Bottom, bottom); }
     void setLeft(const T& left) { setAt(BoxSide::Left, left); }
 
+    RectEdges<T> xFlippedCopy() const
```

Comment:
I'm not really a fan of the names you chose here. We don't put "Copy" in names like this; see my earlier suggestions.